back/conf.js

```diff
@@ -0,0 +1,8 @@
+const  mysql = require('mysql');
+const  database = mysql.createConnection({
+host :  'localhost', // adresse du serveur
+user :  'root', // le nom d'utilisateur
+password :  'jessyjessica', // le mot de passe
+database :  'toshare', // le nom de la base de données
+});
+module.exports = database;
```

Comment:
are you sure you want to push this?

Reply:
absolutely not but I cannot understand why it's pushed because conf.js was added in the gitignore's file.

Reply:
.gitignore's problem regarding conf.js is corrected. 
i pushed the event component with calendar.
we can select a period.